### DISCUSSION TITLE
Removing metabolites in Ham's media (essential task) - Task fail

### DISCUSSION THREAD

**Discussion by beherashr** [SysBioChalmers/Human-GEM#473] [Q&A]
Hi there!
I'm trying to create something like a dropout media experiment on ftINIT - the dropout being, that I remove proline[e] and arginine[e] from the Ham's media. 
I just removed the two amino acids from the metabolicTasks_Essential.txt file and ran prepData, to then continue and create the metabolic models with my transcriptomic data. 
PrepData was successfully created, but the Growth Task failed after I removed proline and arginine from the tasks file (taskReport.ok also showed that the only task that failed was the GR task). 
Do you have an idea as to how I can approach this dropout experiment on ftINIT without the Growth task failing once I remove those amino acids?

**Comment by mihai-sysbio** [comment]
Welcome @beherashr! Just to see if I got this right, you'd like to make the model pretend like an essential task is not really essential?

**Reply by beherashr** [reply]
Thanks for the welcome!
Yes - I wanted to know if the model would still be created if depleted of those two amino acids. Is there a way of making it pretend that it's not essential without entirely changing the Growth task/media?

**Reply by JonathanRob** [reply]
Interesting... since proline and arginine are non-essential amino acids, I would have guessed that the growth task would be ok without them in the media, but maybe there are some other factors I'm forgetting.

What is the output that you want to obtain from the drop-out analysis? If the growth (biomass production) task fails when those two amino acids are removed from the extracellular media, then it will not be possible to achieve growth unless alternative pathways/reactions are added to the model to compensate for their absence. You can remove the task altogether, which would then give you a model without errors, but of course the model would not be able to produce biomass.

**Reply by beherashr** [reply]
Yes, that's what I had expected too. I am running ftINIT with 31 cancer cell lines - in some of those tumors, proline abundancy is high. I want to see if the depletion of the amino acids from the media has some effect on the cancer cell lines, focusing on the Arginine and Proline subsystems reactions (if there are less reactions than with the normal media). 

Theoretically, how can I add alternative reactions to the pathway to compensate? Although this would probably nullify the purpose of the dropout experiment. I don't think removing the task altogether would be an option.

Thank you for your answer!

**Reply by mihai-sysbio** [reply]
It looks like there's more to it than it meets the eye when it comes to the biological essentiality of arginine and proline - they can be essential in certain conditions. Regardless, what matters is their essentiality in the model.

To side-step these amino acids from the media, I'm wondering what's going to happen if the media would contain high levels of certain proteins for which there are degradation reactions (and the corresponding ATP and water). As a starting point, I was looking at the reactions producing these 2 amino acids in the cytosol ([R](https://metabolicatlas.org/explore/Human-GEM/gem-browser/metabolite/MAM01365c), [P](https://metabolicatlas.org/explore/Human-GEM/gem-browser/metabolite/MAM02770c)). Given no other sources of R and P, increased flux through the degradation reactions could indicate that the model is trying to obtain these amino acids.

To elaborate on @JonathanRob's suggestion, you could create a fake new metabolite for which there is minor energy cost to degrade into say both R and P, and then to look at the flux difference through these reactions in the full Ham media and the engineered one.

**Reply by beherashr** [reply]
Yes, I would've expected that maybe the amino acids become essential in tumor tissue - but the task had failed when I ran the prepData with the dropout. 
So how do I enable that the model obtains these amino acids without them being provided extracellularly and how would I add those reactions to the model? 
And what would an example for a "fake" metabolite look like? 

Thanks for the interesting answer!
